jx_sqlite/insert_table.py

```diff
@@ -281,6 +282,28 @@ def _flatten(data, uid, parent_id, order, full_path, nested_path, row=None, guid
                 elif c.type=="nested" and value_type=="object":
                     value_type = "nested"
                     v = [v]
+                elif len(c.nested_path) < len(nested_path):
+                    from_doc = doc_collection.get(c.nested_path[0], None)
+                    column = c.es_column
+                    from_doc.active_columns.remove(c)
+                    abs_schema.remove(cname, c)
+                    required_changes.append({"nest": (c, nested_path[0])})                    
+                    c = Column(
```

Comment:
do not reuse the name `c`, call it `deep_c`, or `deeper_c`, or `new_c`.  Then, you can see both the new and old values when debugging